src/cloud/docker/docker-extend.md

```diff
@@ -27,4 +27,22 @@ You can use the built-in extension mechanism of Docker to specify [multiple comp
    docker-compose -f docker-compose.yml -f docker-compose-dev.yml run deploy bash
    ```
 
+## Specify Docker Build Sources
+To be able to locally test changes to images or if there is a requirement to make more extensive changes to the containers you will need to build them from source. This is accomplished by adding build context to the `docker-compose.override.yml`.
+
+```yaml
+version: '2.1'
+services:
+ web:
+   build:
+     context: ./vendor/magento/magento-cloud-docker/images/nginx/1.9/
+```
+This example defines the build context for the web container, however this can be done with any of the images in `vendor/magento/magento-cloud-docker` or any image, including locally resourced outside of the project.
```

Comment:
The suggestion above moves this sentence suggest to the paragraph introducing the example.

Wasn't sure what you mean by "locally resourced outside the project"--is there a clearer way to say that.  I think you mean if you have a Docker image that you've customized locally, you can also build it from source?